src/Views/Appearance.vala

```diff
@@ -76,6 +92,51 @@ public class Appearance : Gtk.Grid {
 
         update_text_size_modebutton (interface_settings);
 
+        Pantheon.AccountsService? pantheon_act = null;
+        FDO.Accounts? accounts_service = null;
+        string? user_path = null;
+        try {
+            accounts_service = GLib.Bus.get_proxy_sync (
+                GLib.BusType.SYSTEM,
+               "org.freedesktop.Accounts",
+               "/org/freedesktop/Accounts"
+            );
+
+            user_path = accounts_service.find_user_by_name (GLib.Environment.get_user_name ());
+        } catch (Error e) {
+            critical (e.message);
+        }
+
+        try {
+            pantheon_act = GLib.Bus.get_proxy_sync (
+                GLib.BusType.SYSTEM,
+                "org.freedesktop.Accounts",
+                user_path,
```

Comment:
After getting the user path, you should probably check if it's not an empty string before passing it to another DBus method.

Reply:
@donadigo if it's an empty string, what should we do?